examples/visualisation/visualise_data.ipynb

```diff
@@ -19,13 +21,64 @@
     "### `visualization`\n",
     "contains utilities to draw additional information (e.g. trajectories) onto RGB images. These utilities are commonly used after a `to_rgb` call to add other information to the final visualisation. \n",
     "One example is:\n",
-    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n"
+    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n",
+    "\n",
+    "*By using L5 datasets, you accept the license at https://self-driving.lyft.com/level5/download/ and the licenses present inside of each dataset archive.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "-2bOYHKU1tw2"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Download L5 Sample Dataset and install L5Kit\n",
+    "!rm -rf l5kit_dataset\n",
+    "\n",
+    "# Download sample zarr\n",
+    "print(\"Downloading sample zarr dataset...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/sample.tar -q --show-progress\n",
+    "!tar xf sample.tar\n",
+    "!rm -rf sample.tar\n",
+    "!mkdir -p l5kit_dataset/scenes\n",
+    "!mv sample.zarr l5kit_dataset/scenes\n",
+    "\n",
+    "# Download semantic map\n",
+    "print(\"Downloading semantic map...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/semantic_map.tar -q --show-progress\n",
+    "!mkdir -p l5kit_dataset/semantic_map\n",
+    "!tar xf semantic_map.tar -C l5kit_dataset/semantic_map\n",
+    "!rm -rf semantic_map.tar\n",
+    "\n",
+    "# Download aerial maps\n",
+    "print(\"Downloading aerial maps (this can take a while)...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/aerial_map.tar -q --show-progress\n",
+    "!tar xf aerial_map.tar\n",
+    "!mv aerial_map l5kit_dataset\n",
+    "!rm -rf aerial_map.tar\n",
```

Comment:
don't we need also to move  `meta.json` into the root folder ? It should be inside `semantic_map.tar`

Reply:
The semantic map is already being decompressed directly into the destination folder. Is that what you mean ?

Reply:
isn't it decompress inside `l5kit_dataset/semantic_map` ?

Reply:
Yes, everything from the `semantic_map.tar` will go into `l5kit_dataset/semantic_map`.

Reply:
ah I see what's happening then!

I think we're falling back to [this](https://github.com/lyft/l5kit/blob/19b7ac8a35f50467ee8916b58e60298b5f5e7146/l5kit/l5kit/rasterization/rasterizer_builder.py#L144) or [this](https://github.com/lyft/l5kit/blob/19b7ac8a35f50467ee8916b58e60298b5f5e7146/l5kit/l5kit/rasterization/rasterizer_builder.py#L161) then

I've checked and the `meta.json` is not where it should be in fact...this could become an issue once we remove the hard-coded matrices

Reply:
Wasn't aware of these matrices there, I think we should add another PR adding warnings upon fallback, I will open a new issue for us to keep track.

Reply:
ideally we should remove the fallback entirely. The dataset has been updated now